### ISSUE TITLE
Screen reader tools (JAWS, VoiceOver) doesn't read Combobox or multi select box options using Arrow keys 

### ISSUE THREAD

**Issue by joejeet** [carbon-design-system/carbon-components-vue#1090]
#### When using a screen reader tool, using the arrow keys it is not reading the options in (Chrome-Mac using VoiceOver) and (Chrome-Windows using JAWS)

> Is this a feature request (new component, new icon), a bug, or a general issue? **Bug**

> Is this issue related to a specific component? **CvMultiSelect, CvComboBox**

> What did you expect to happen? What happened instead? What would you like to see changed?
> #### I expect it to work like React Combobox carbon components it works https://react.carbondesignsystem.com/iframe.html?id=combobox--combobox

> What browser are you working in? **Latest Chrome**

> What version of the Carbon Design System are you using?    **"^10.24.0"**

## Steps to reproduce the issue
Using VoiceOver or JAWS tool and 
1. http://vue.carbondesignsystem.com/iframe.html?id=components-cvmultiselect--default
2. http://vue.carbondesignsystem.com/iframe.html?id=components-cvcombobox--default

## Additional information

- For React  Combobox carbon components it works https://react.carbondesignsystem.com/iframe.html?id=combobox--combobox
